scm/src/gmtb_scm_input.f90

```diff
@@ -77,16 +80,19 @@ subroutine get_config_nml(scm_state)
   runtime = 2138400.0
   output_frequency = 600.0
   n_levels = 64
+  n_soil   = 4
```

Comment:
Note to self: n_soil is "hard-coded" here and is not a namelist variable. This will likely need to change since the SCM could be used with the RUC LSM which uses 9 soil levels.

Reply:
good point.  guess that can be added to the experiment namelist. Want me to do that?

Reply:
No, I can do that in a followup PR. It might be nice to do something fancier too like autopopulate its value based on the land surface scheme in the CCPP SDF.

Reply:
NEMSfv3gfs uses Model%lsm, Model%lsm_{noah,noahmp,ruc} = {1,2,3} for this purpose.